src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -160,6 +199,38 @@ public function enterNode(Node $node) : ?int
                     return NodeTraverser::DONT_TRAVERSE_CHILDREN;
                 }
 
+                if ($node instanceof Node\Stmt\Const_) {
+                    foreach ($node->consts as $constNode) {
+                        /** @psalm-suppress UndefinedPropertyFetch */
+                        $this->constantNodes[$constNode->namespacedName->toString()] = $node;
+                    }
+
+                    return NodeTraverser::DONT_TRAVERSE_CHILDREN;
+                }
+
+                if ($node instanceof Node\Expr\FuncCall
+                    && $node->name instanceof Node\Name
+                    && $node->name->toLowerString() === 'define'
+                ) {
+                    /** @psalm-suppress UndefinedPropertyFetch */
+                    $constantName = $node->args[0]->value->value;
+
+                    // Not supported because of resource as value
+                    if (in_array($constantName, ['STDIN', 'STDOUT', 'STDERR'], true)) {
```

Comment:
it's fine to not support it but as of now it's bailing out with a `TypeError` on https://github.com/Roave/BetterReflection/pull/471/files#r284000358 due to this

Reply:
Should be fixed.